src/api.js

```diff
@@ -123,3 +124,25 @@ export async function getSingleMovieReview(
     return reviewArray;
   }
 }
+
+// data = array of screenings, today = todays time and date in ISO format
+export function filterOutOldScreenings(data, today){
+ 
+  // run through the array and return objects containing time >= todaysTime
+  const res = data.filter(comming);
+    
+  // currentValue is reqired parameter but not needed, index is index of current object
+      function comming(currentValue, index){
+        const screening = new Date(data[index].attributes.start_time);
```

Comment:
Instead of `data[index]` you could use `currentValue`, which would be better in many ways:

* It's a more usual way to write these functions, so semi-experienced programmers would have an easier time reading the code
* It means the function can be used regardless of what array you use it on, not just the array called `data`